mesmerize_viz/_cnmf.py

```diff
@@ -907,15 +897,14 @@ def set_component_index(self, index):
 
         self._component_metrics_text.value = metrics
 
-    def _zoom_into_component(self, index: int):
-        for subplot in self._image_widget.figure:
-            subplot.camera.show_object(
-                subplot["contours"].graphics[index].world_object,
-                scale=self.zoom_components_scale.value
-            )
+    def _set_zoom_flag(self, value: bool):
```

Comment:
can we please call this something more informative than "flag", perhaps something like "_set_auto_zoom"